static/js/containers/PostPage_test.js

```diff
@@ -259,69 +277,68 @@ describe("PostPage", function() {
+    assert.isTrue(inner.find(ExpandedPostDisplay).props().showPermalinkUI)
   })
 
   it("should hide the comments header section when there are no comments", async () => {
     post.num_comments = 0
     helper.getPostStub.returns(Promise.resolve(post))
     helper.getCommentsStub.returns(Promise.resolve([]))
-    const wrapper = await renderPage()
-    assert.isFalse(wrapper.find(".count-and-sort").exists())
+    const { inner } = await render()
+    assert.isFalse(inner.find(".count-and-sort").exists())
   })
 
   //
   ;[true, false].forEach(userIsAnon => {
     it(`should show a ReplyToPostForm when userIsAnonymous() === ${userIsAnon}`, async () => {
       const anonStub = helper.sandbox.stub(utilFuncs, "userIsAnonymous")
       anonStub.returns(userIsAnon)
-      const wrapper = await renderPage()
-      wrapper.update()
-      assert.ok(wrapper.find(ReplyToPostForm).exists())
+      const { inner } = await render()
+      assert.ok(inner.find(ReplyToPostForm).exists())
     })
   })
 
   it("passed props to each CommentVoteForm", async () => {
-    const wrapper = await renderPage()
-    const commentTree = wrapper.find("CommentTree")
+    const { inner } = await render()
+    const commentTree = inner.find("CommentTree")
     const commentTreeProps = commentTree.props()
-    for (const form of wrapper.find("CommentVoteForm")) {
+    for (const form of inner.find("CommentVoteForm")) {
       const fromProps = form.props
       assert.equal(fromProps.downvote, commentTreeProps.downvote)
       assert.equal(fromProps.upvote, commentTreeProps.upvote)
     }
   })
 
   it("passed props to each CommentRemovalForm", async () => {
-    const wrapper = await renderPage()
-    const commentTree = wrapper.find("CommentTree")
+    const { inner } = await render()
+    const commentTree = inner.find("CommentTree")
     const commentTreeProps = commentTree.props()
-    for (const form of wrapper.find("CommentRemovalForm")) {
+    for (const form of inner.find("CommentRemovalForm")) {
       const fromProps = form.props
       assert.equal(fromProps.approve, commentTreeProps.approve)
       assert.equal(fromProps.remove, commentTreeProps.remove)
     }
   })
 
   it("loads more comments when the function is called", async () => {
-    const wrapper = await renderPage()
-    const commentTree = wrapper.find("CommentTree")
+    const { inner, store } = await render()
+    const commentTree = inner.find("CommentTree")
     const commentTreeProps = commentTree.props()
     const parent = comments[0]
     const moreComments = makeMoreComments(post, parent.id)
     const newComment = makeComment(post, parent.id)
     const newComments = [newComment]
 
     helper.getMoreCommentsStub.returns(Promise.resolve(newComments))
-    await listenForActions(
+    await commentTreeProps.loadMoreComments(moreComments)
+    const actionsList = store.getActions()
+    assert.deepEqual(
+      actionsList.slice(actionsList.length - 3).map(action => action.type),
```

Comment:
I think you can rewrite this with [`R.pluck`](https://ramdajs.com/docs/#pluck) and [`assert.includeMembers`](https://www.chaijs.com/api/assert/#method_includemembers). Something like:

```javascript
assert.includeMembers(
  R.pluck("type")(actionsList),
  [
    actions.morecomments.get.requestType,	        
    actions.morecomments.get.successType,
    REPLACE_MORE_COMMENTS
  ]
)
```